### DISCUSSION TITLE
FLUX on 3060, optimal version?

### DISCUSSION THREAD

**Discussion by srg-vlcv** [lllyasviel/stable-diffusion-webui-forge#1584] [General]
Hey. Can you tell me about FLUX: I use Forge, ComfUI is complicated and inconvenient for me. Right now I am using bnb nf4, video card 3060 12gb. Question: what version of FLUX can I rely on to get my video card fully up and running? For example I used Q4_1 for a while, downloaded Q5_K_S and it didn't work. The error was always related to mat1 and mat2 and I couldn't solve the problem. Went back to nf4 again, and there is a feeling that this is not the optimal model for me.
P.S. I'm still a little “swimming” in this subject, do not judge harshly

**Comment by ckotty-alex** [comment]
I have same 3060/12 and 32gb of RAM, flux1-dev-q8_0.gguf works well

**Reply by srg-vlcv** [reply]
> I have same 3060/12 and 32gb of RAM, flux1-dev-q8_0.gguf works well

Thanks! do you load ae, clip and t5 the same way? I have 16gb RAM but a large swap file

**Reply by ckotty-alex** [reply]
> > I have same 3060/12 and 32gb of RAM, flux1-dev-q8_0.gguf works well
> 
> Thanks! do you load ae, clip and t5 the same way? I have 16gb RAM but a large swap file

Yes, I do. Just try it, it's a bit slowly than nf4 and  gguf checkpoints, but has better quality, imho

**Reply by srg-vlcv** [reply]
> > > I have same 3060/12 and 32gb of RAM, flux1-dev-q8_0.gguf works well
> > 
> > 
> > Thanks! do you load ae, clip and t5 the same way? I have 16gb RAM but a large swap file
> 
> Yes, I do. Just try it, it's a bit slowly than nf4 and gguf checkpoints, but has better quality, imho

![image](https://github.com/user-attachments/assets/4ca4fbee-8114-49ad-bd07-5c473453f7d7)
Hi again! Listen, since you use forge, can you tell me how to make files saved not just under numbers, but with additional information? There is such a setting, I turned it on, but it didn't help.

**Reply by camelos** [reply]
> > > > I have same 3060/12 and 32gb of RAM, flux1-dev-q8_0.gguf works well
> > > 
> > > 
> > > Thanks! do you load ae, clip and t5 the same way? I have 16gb RAM but a large swap file
> > 
> > 
> > Yes, I do. Just try it, it's a bit slowly than nf4 and gguf checkpoints, but has better quality, imho
> 
> ![image](https://private-user-images.githubusercontent.com/168664863/363433598-4ca4fbee-8114-49ad-bd07-5c473453f7d7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyMTgwMzQsIm5iZiI6MTcyNTIxNzczNCwicGF0aCI6Ii8xNjg2NjQ4NjMvMzYzNDMzNTk4LTRjYTRmYmVlLTgxMTQtNDlhZC1iZDA3LTVjNDczNDUzZjdkNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwMVQxOTA4NTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01M2Q2NjU3NGRkZDY0ZTgwMTVjZDk0MTAyNGVhYThmYmQ4MWRiYzM2NWY1Yjk4OGU3NjMyZTk3ZWRmMWI0ZGM3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.0_tyW2MwFQj76fDGu402rTG-qnJ9s-Sq8HpHTKDDWEo) Hi again! Listen, since you use forge, can you tell me how to make files saved not just under numbers, but with additional information? There is such a setting, I turned it on, but it didn't help.

Settings - Saving images/grids - [[wiki] Images filename pattern (see wiki - https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Custom-Images-Filename-Name-and-Subdirectory)
